app/controllers/admin/content_controller.rb

```diff
@@ -37,6 +37,24 @@ def edit
     new_or_edit
   end
 
+  def merge
+    if current_user.admin?
```

Comment:
Great suggestion - updated it so that a non-admin gets a flash notice and then is redirected to the index page.